CONTRIBUTING.md

```diff
@@ -75,41 +78,41 @@ More examples can be found [here](https://www.conventionalcommits.org/en/v1.0.0/
 
-Before contributing please read and approve [our Code Of Conduct](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/CODE_OF_CONDUCT.md) outlining our shared values and expectations. 
+Before contributing, please read and approve [our Code Of Conduct](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/blob/main/CODE_OF_CONDUCT.md) outlining our shared values and expectations. 
 
 ## Building the Dynatrace Monitoring as Code Tool
 
 The `monaco` tool is written in [Go](https://golang.org/), so you will need to have [installed Go](https://golang.org/dl/) to build it.
 
-To build the tool run `make build` in the repository root folder. 
+To build the tool, run `make build` in the repository root folder. 
 
 > This guide references the make target for each step. If you want to see the actual Go commands take a look at the [Makefile](./Makefile)
 
-To install the tool to your machine run `make install` in the repository root folder.
+To install the tool to your machine, run `make install` in the repository root folder.
 
 This will create a `monaco` executable you can use.
 
 To build a platform specific executable run: `GOOS={OS} GOARCH={ARCH} make build`.
 
-For example a Windows executable can be built with `GOOS=windows GOARCH=386 make build BINARY=monaco.exe`.
+For example, a Windows executable can be built with `GOOS=windows GOARCH=386 make build BINARY=monaco.exe`.
 
 ## Testing the Dynatrace Monitoring as Code Tool
 
 Run the unit tests for the whole module with `make test` in the root folder.
 
 ### Integration Tests
 
-In addition to unit tests, the module contains integration tests, that upload configuration to two test environments. Those are tagged `integration` and will be run for any pull request opened for Monitoring as Code.
+In addition to unit tests, the module contains integration tests that upload configuration to two test environments. Those are tagged `integration` and will be run for any pull request opened for Monitoring as Code.
 
-To run the integration tests you will need at least one Dynatrace environment - the tests run against two configurable environments.
+To run the integration tests, you will need at least one Dynatrace environment - the tests run against two configurable environments.
 
-The following environment variables need to be defined for these test environments:
+Define the following environment variables to test for these environments:
 * `URL_ENVIRONMENT_1` ... URL of the first test environment
 * `TOKEN_ENVIRONMENT_1` ... API token for the first test environment 
 * `URL_ENVIRONMENT_2` ... URL of the second test environment
 * `TOKEN_ENVIRONMENT_2` ... API token for the second test environment 
 
-The integration tests can be run with `make integration-test`
+Run the integration tests using `make integration-test`.
 
 For convenience single package tests can be run with `make test-package pkg={PACKAGE}` - e.g. `make test-package pkg=api`.
```

Comment:
unrelated, but while you're at it: could you move this up into the testing section around line 99 please? test-package runs unit tests only